.travis.yml

```diff
@@ -1,25 +1,37 @@
-language: c
+language: cpp
```

Comment:
Is that required ? Changing the whole project from C to C++ (also in cmake) just for opjstyle seems a bit overkill. No other option ?